packages/combo-box/test/scrolling.test.js

```diff
@@ -1,30 +1,19 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync, focusout, isIOS } from '@vaadin/testing-helpers';
+import { fixtureSync, focusout, nextRender } from '@vaadin/testing-helpers';
 import './not-animated-styles.js';
 import '../vaadin-combo-box.js';
 
 describe('scrolling', () => {
   let comboBox, overlay, scroller, input;
 
-  beforeEach(() => {
+  beforeEach(async () => {
     comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+    await nextRender();
     overlay = comboBox.$.overlay;
     input = comboBox.inputElement;
     scroller = comboBox._scroller;
   });
 
-  afterEach(() => {
-    comboBox.close();
-  });
```

Comment:
This isn't needed since we have `disconnectedCallback` in `ComboBoxMixin` that handles overlay cleanup.